code/src/App.js

```diff
@@ -1,12 +1,27 @@
 import React from 'react'
 import data from './data.json'
+import AlbumCover from "./Components/AlbumCover"
+import styles from './styles'
+import Radium from 'radium'
 
-console.log(data)
-
-export const App = () => {
-  return (
-    <div>
-      Find me in src/app.js!
+const App = () => (
+  <div style={styles.wrapper}>
+    <h1 style={styles.h1}>New albums and singles</h1>
+    <div style={styles.flex("space-between")}>
+      {data.albums.items.map(album => (
+        <div key={album.id} style={styles.albumCoverWrapper}>
+          <AlbumCover
+            image={album.images[1].url}
+            name={album.name}
+            artist={album.artists[0].name}
+            linkAlbum={album.external_urls.spotify}
+            linkArtist={album.artists[0].external_urls.spotify}
```

Comment:
Regarding the artists.
Since some of the albums have multiple artists you should make sure to show all of them.
Pass along the whole artists-array as a prop, so instead of artist and linkArtist go for artists={album.artists}